### PR TITLE
Comment for cultural heritage items

### PR DESCRIPTION
**Pull request by kutaycandan**
Changes proposed in this pull request: 
- Comment create is added

### CODE REVIEW

File: Atlas/backend/atlas/urls.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
     url(r'^cultural_heritage_item/(?P<heritage_id>[0-9]+)/image/?$', views.image_media_item.as_view()),
     url(r'^cultural_heritage_item/(?P<id>\d+)/?$', views.cultural_heritage_item_view_update_delete.as_view()),
     url(r'^cultural_heritage_item/myitems', views.cultural_heritage_item_list_user_items.as_view()),
+    url(r'^comment/', views.comment.as_view()),
```
Comment (YmirKhang) [inline Atlas/backend/atlas/urls.py]: This url must be like Cultural_heritage_item/<Number of cultural heritage item>/comments for all comments on a spesific item. And one more url like /cultural_heritage_item/<id of cultural heritage item>/<id of comment> to retrieve or update a spesific comment

File: Atlas/backend/atlas/views.py [hunk 2]
```diff
@@ -29,6 +29,10 @@ def create(self, request, *args, **kwargs):
 
         return super(cultural_heritage_item, self).create(request,*args,**kwargs)
 
+class comment(generics.ListCreateAPIView):
```
Comment (YmirKhang) [inline Atlas/backend/atlas/views.py]: There must be a queryset to retrieve all the comments on a spesific item

**Review by YmirKhang — COMMENTED** [review]
Would be cool if the points I have commented on can be changed.

File: Atlas/backend/atlas/views.py (hunk 2 above)
Comment (kutaycandan) [reply Atlas/backend/atlas/views.py]: Okey, I'll work on it

File: Atlas/backend/atlas/urls.py (hunk 1 above)
Comment (SaitTalhaNisanci) [reply Atlas/backend/atlas/urls.py]: Yeah, basically you need to use cultural_heritage_item/id/comment to create a new comment for cultural heritage item with id 'id'. You will use the same as I did for item endpoint.  ```url(r'^cultural_heritage_item/(?P<heritage_id>[0-9]+)/comment/?$',``` That way you can find the corresponding cultural heritage item from the reviews.

**Review by SaitTalhaNisanci — CHANGES_REQUESTED** [review]

**Comment by SaitTalhaNisanci** [comment]
Also, you need to add tests for adding comments to a cultural heritage item. Without tests we cant know for sure if this is working, or even if we do, we cant know in the future if it will work after changing some other code related to this.

**Pull request closed without merging**